README.md

```diff
@@ -1,25 +1,32 @@
+# Domain Model for a School
 
+## Objectives
 
-# Domain Model for a School
+1. Define the concept of a domain model. 
+2. Create your own domain model. 
+3. Write instance methods that manipulate nested data structures. 
+
+## What is a Domain Model?
 
-In this assignment you'll be writing a simple app for a school. 
+A domain model is a representation of real-world concepts in software. The concept of domain modeling is key in object orientation. In object orientation, we think of our classes as templates for objects. The instances of our classes are thought of as objects. For example, a Person class produces people objects that have attributes and behaviors, as described and enacted by instance methods. 
 
-Create the app detailed below and get the tests passing. 
+Let's think of an example on a slightly larger scale. Let's say we are writing an application that will be used by a major automobile manufacturer to help manage their plants. In this case, we need a program that represents not just individual cars, but the entire car factory. In this (simplified) example, you could write a `AutoFactory` class that produces individual cars and has instance and class methods that handle things like `take_inventory` or `paint_cars`. Such a program not only represents a single object or concept but an entire environment––that of the auto factory. Consequently, this program could be understood as a domain model. It takes the world of the auto factory, and maps the constituents of that world into your program. 
```

Comment:
okay yea that makes sense. I meant "factory" like a real-life factory. point of example is to have them stat thinking of their programs as modeling real-world environments (like "school"). Domain model should more appropriately encompass more than one class. I will add something to that effect and also re-name this class to something like `AutoPlant`. Let me know how that sounds.